graphql/resolvers.ts

```diff
@@ -53,12 +64,62 @@ export default {
         submissions,
         lessonStatus
       }
+    },
+    async isTokenValid(_parent: void, args: { cliToken: string }) {
+      const { cliToken } = args
```

Comment:
move into `authController`. 

1. base64 decode cliToken, JSON.parse to get inputObj: `{id: 5, cliToken: 'faketoken123'}`
2. `User.findOne( {id})`